crates/ruma-client-api/src/r0/keys/upload_signing_keys.rs

```diff
@@ -4,7 +4,9 @@
 
 use ruma_api::ruma_api;
 
-use super::CrossSigningKey;
+#[cfg(feature = "unstable-pre-spec")]
```

Comment:
This attribute needs to be removed, otherwise the crate will fail to compile without the unstable-pre-spec feature.

Reply:
Seems like we also need to adjust CI to catch stuff like this.

Reply:
Also there should be no empty line between the ruma_api and ruma_common imports.